XbSymbolDatabase.c

```diff
@@ -478,21 +484,21 @@ void XbSymbolScanOOVPA(OOVPATable *OovpaTable,
         }
 
         // Skip higher build version
-        if (buildVersion < pLoop->Version)
+        if (bStrictRevisionLimit && buildVersion < pLoop->Version)
             continue;
 
         // Search for each function's location using the OOVPA
         xbaddr pFunc = (xbaddr)(uintptr_t)XbSymbolLocateFunction(pLoop->Oovpa, lower, upper, (uintptr_t)xb_start_virt_addr);
         if (pFunc == 0)
             continue;
 
-#if 0 // TODO: Find a way to support this.
         if (pFunc == pLastKnownFunc && pLastKnownSymbol == pLoop - 1) {
-            if (g_SymbolAddresses[pLastKnownSymbol->szFuncName] == 0) {
-                printf("HLE: Duplicate OOVPA signature found for %s, %d vs %d!\n", pLastKnownSymbol->szFuncName, pLastKnownSymbol->Version, pLoop->Version);
-            }
+            //if (g_SymbolAddresses[pLastKnownSymbol->szFuncName] == 0) {
+                char output[2048];
+                sprintf(output, "Duplicate OOVPA signature found for %s, %hd vs %hd!", pLastKnownSymbol->szFuncName, pLastKnownSymbol->Version, pLoop->Version);
+                XbSymbolOutputMessage(XB_OUTPUT_MESSAGE_WARN, output);
```

Comment:
Why doesn't this use `output_func` ?

Reply:
Resolved, [see my other comment](https://github.com/Cxbx-Reloaded/XbSymbolDatabase/pull/60#discussion_r257467231)